docs/releases/1.8.3.txt

```diff
@@ -50,3 +50,6 @@ Bugfixes
 
 * Fixed recording of applied status for squashed (replacement) migrations
   (:ticket:`24628`).
+
+* Fixed prefetch_related for models using UUID primary keys on non-PostgreSQL
```

Comment:
We don't usually document bug fixes in major release notes, is there something else you were thinking about?